include/wx/qt/dvrenderers.h

```diff
@@ -34,14 +34,16 @@ class WXDLLIMPEXP_ADV wxDataViewTextRenderer: public wxDataViewRenderer
 class WXDLLIMPEXP_ADV wxDataViewBitmapRenderer: public wxDataViewRenderer
 {
 public:
-    static wxString GetDefaultType() { return wxS("wxBitmap"); }
+    static wxString GetDefaultType() { return wxS("wxBitmapBundle"); }
 
     wxDataViewBitmapRenderer( const wxString &varianttype = GetDefaultType(),
                               wxDataViewCellMode mode = wxDATAVIEW_CELL_INERT,
                               int align = wxDVR_DEFAULT_ALIGNMENT );
 
     bool SetValue( const wxVariant &value );
     bool GetValue( wxVariant &value ) const;
+
+    bool IsCompatibleVariantType(const wxString& variantType) const;
```

Comment:
This would result in "inconsistent use of override" warnings as it's not used for the other methods of the same class, so omitting it looks like a lesser evil (of course, ideal would be to add it everywhere but considering this is nothing more than a stub anyhow, I didn't feel like doing it would be useful).